tests/ltp/Makefile

```diff
@@ -36,6 +36,9 @@ $(ROOT_FS): $(ALPINE_TAR) buildenv.sh
 	$(ESCALATE_CMD) install buildenv.sh $(MOUNTPOINT)/usr/sbin
 	$(ESCALATE_CMD) chroot $(MOUNTPOINT) /sbin/apk update
 	$(ESCALATE_CMD) chroot $(MOUNTPOINT) /sbin/apk add bash
+	$(ESCALATE_CMD) mkdir $(MOUNTPOINT)/ltp_tst_mnt_fs
```

Comment:
This has to be freshly created for each test case. @hukoyu Is that right? I think that's the plan, but I lost track a bit.

Reply:
@letmaik we create an image and then we use a fresh copy of this image for each LTP test. We don't create a new image for each test but we keep the original image and copy it as fresh image for each test.

Reply:
Ah alright, and this image would contain the disk which is created here.

Reply:
@SeanTAllen  already submitted https://github.com/lsds/sgx-lkl/pull/746 to combine make files for ltp.